example/timer/odp_timer_accuracy.c

```diff
@@ -874,6 +874,19 @@ static void run_test(test_global_t *test_global)
 		ctx = odp_timeout_user_ptr(tmo);
 		tmo_ns = ctx->nsec;
 		if (mode == MODE_PERIODIC) {
+			if (!ctx->count && !test_global->opt.offset_ns) {
+				/*
+				 * If first_tick is zero, the API allows the implementation to
+				 * place the timer where it can, so we just take the first timeout
+				 * as a baseline, if it's within a reasonable range.
+				 */
+				int64_t diff = (int64_t)time_ns - (int64_t)tmo_ns;
+
+				if (diff < 0)
+					diff *= -1;
+				if (diff < period_dbl / 2)
```

Comment:
This would be simpler than multiply by -1:
 diff = -diff;
 
 Also, "period_dbl / 2" could be defined as a const variable to make the error margin more visible:
 
const int64_t error_margin = 0.5*period_dbl;

Reply:
In v2, I removed these conditions and instead we print the diff. There are other changes as well, involving odp_timeout_tick(). Please re-review.